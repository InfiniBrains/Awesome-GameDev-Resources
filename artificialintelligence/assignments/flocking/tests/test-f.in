1.000 0.000 0.000 1.000 1.000 0.000 2.000 2
0.000 0.500 -1.000 2.000
0.000 -0.500 3.000 1.000
0.125
