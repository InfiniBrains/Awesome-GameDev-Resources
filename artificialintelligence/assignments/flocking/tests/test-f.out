-0.094 0.781 -0.750 2.250
0.406 -0.312 3.250 1.500
